tw/fn.go

```diff
@@ -245,9 +245,9 @@ func IsNumeric(s string) bool {
 	return err == nil
 }
 
-// SplitCamelCase splits a camelCase or PascalCase string into separate words.
+// SplitCase splits a camelCase or PascalCase or snake_case string into separate words.
 // It detects transitions between uppercase, lowercase, digits, and other characters.
-func SplitCamelCase(src string) (entries []string) {
+func SplitCase(src string) (entries []string) {
```

Comment:
Is there any reason why you changed the name , since this is a public function then more breaking changes

Reply:
we can till change it to SplitCase but we need to ensure `SplitCamelCase`  is not removed but with deprecation

Reply:
You're right we should not change this name. I changed it as it supports now not only CamelCase, PascalCase but also snake case. I'll revert name change